### ISSUE TITLE
webui: doesn't fully load when using a binary from `make buildoss`

### ISSUE THREAD

**Issue by rkruze** [cockroachdb/cockroach#34117]
The webui console doesn't fully load and only shows the following in the source: 

```<!DOCTYPE html>
<html>
	<head>
		<title>Cockroach Console</title>
		<meta charset="UTF-8">
		<link href="favicon.ico" rel="shortcut icon">
	</head>
	<body>
		<div id="react-layout"></div>

		<script>
			window.dataFromServer = {"ExperimentalUseLogin":true,"LoginEnabled":false,"LoggedInUser":null,"Tag":"v2.1.3","Version":"v2.1"};
		</script>

		<script src="protos.dll.js" type="text/javascript"></script>
		<script src="vendor.dll.js" type="text/javascript"></script>
		<script src="bundle.js" type="text/javascript"></script>
	</body>
</html>

```

**To Reproduce**

1. Checkout v2.1.3 tag
2. Run `make buildoss`
3. Run the cockroachoss binary like `cockroachoss start --insecure`
4. Go to the webui - http://localhost:8080

**Expected behavior**
I would expect most of the WebUI to load

**Environment:**
 - CockroachDB version 2.1.3
 - Server OS: Mac
 - Client app NA

**Comment by knz** [comment]
cc @couchand @piyush-singh for triage. Maybe @petermattis if there's a makefile bug.

**Comment by riggs-** [comment]
I can confirm the same problem on FreeBSD and version 2.1.4. Will try to find some more info as soon as I can.

**Comment by knz** [comment]
This is fixed in v21.1
